sdk/azcore/README.md

```diff
@@ -20,6 +20,83 @@ go get github.com/Azure/azure-sdk-for-go/sdk/azcore
 
 General documentation and examples can be found on [pkg.go.dev](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azcore).
 
+## Key concepts
+
+The main shared concepts of `azcore` include:
+
+- Accessing HTTP response details for the returned model of any SDK client operation.
+- Polling long-running operations (LROs), via `Poller`.
+- Pagenation for pageable apis, via `Pager`.
+- Abstractions for Azure SDK credentials (`TokenCredential`).
+- HTTP pipeline and HTTP policies such as retry and logging, which are configurable via service client specific options.
+
+### Long Running Operations
+
+Some operations take a long time to complete and require polling for their status. Methods starting long-running operations return `Poller[T]` types.
+
+You can intermittently poll whether the operation has finished by using the `Poll()` method on the returned `Poller[T]`.  Alternatively, if you just want to wait until the operation completes, you can use `PollUntilDone()`.
+
+
+#### PollUntilDone Example
+```go
+client, err := exampleservice.NewClient()
+
+poller, err := client.SomeLongRunningAPI()
+if err != nil {
+    panic(err)
+}
+
+pollResp, err := poller.PollUntilDone(context.TODO())
+if err != nil {
+    panic(err)
+}
+// Do something with the result
+//  i.e. fmt.Printf("Retrieved value %s", *pollResp.Value.Property)
+
+```
+
+#### Poll Example
+```go
+client, err := exampleservice.NewClient()
+
+poller, err := client.SomeLongRunningAPI()
+if err != nil {
+    panic(err)
+}
+
+for {
+    _, err := poller.Poll()
+    if err != nil {
+        panic(err)
+    }
+    if poller.Done(){
+        result, err != poller.Result(context.TODO())
+        if err != nil {
+            panic(err)
+        }
+        //Do Something with the result
+        break
+    }
+}
+```
+### Consuming Service Methods Returning `Pager`
+
+If a service call returns multiple values in pages, it would return a `Pager` as a result.  You can iterate over pages using the following coding pattern:
```

Comment:
```suggestion
If a service call returns multiple values in pages, it would return a `runtime.Pager` as a result.  You can iterate over pages using the following coding pattern:
```